lib/browser/api/web-contents.ts

```diff
@@ -148,23 +148,23 @@ WebContents.prototype._sendInternal = function (channel, ...args) {
 
   return this._send(true /* internal */, channel, args);
 };
-WebContents.prototype.sendToFrame = function (frameId, channel, ...args) {
+WebContents.prototype.sendToFrame = function (frame, channel, ...args) {
   if (typeof channel !== 'string') {
     throw new Error('Missing required channel argument');
-  } else if (typeof frameId !== 'number') {
-    throw new Error('Missing required frameId argument');
+  } else if (!(typeof frame === 'number' || Array.isArray(frame))) {
```

Comment:
extract to a helper function maybe?

Reply:
The C++ side will also catch this, so I don't think an extra check is needed. In fact this check isn't really needed either, but I'll leave it because it makes the diff smaller to keep it.